src/docs/tutorial.tex

```diff
@@ -459,11 +459,13 @@ \section{Booleans: {\tt bool}}
 boolean value is true:
 
 \begin{minted}{felix}
-assert$ 1 < 2;
+assert 1 < 2;
 \end{minted}
 
 If the argument of an \verb%assert% is false, then if control
 flows through it, the program is terminated with an error message.
+Note that as far as \verb%assert% is neither procedure nor function
```

Comment:
This reads oddly to me. How about:

> Note that, because \verb%assert% is a keyword, not a procedure, you do not need to fix operator priorities with \verb%$%.